README.md

```diff
@@ -343,315 +375,340 @@ convert asff2hdf            Translate a AWS Security Finding Format JSON into a
+    -o, --output=<value>  (required) Output HDF JSON File
 
   EXAMPLES
-    saf convert scoutsuite2hdf -i scoutsuite-results.js -o output-hdf-name.json
+    $ saf convert scoutsuite2hdf -i scoutsuite-results.js -o output-hdf-name.json
 ```
 
-
 ##### Snyk to HDF
-
 ```
-convert snyk2hdf             Translate a Snyk results JSON file into a Heimdall
-                             Data Format JSON file
-  OPTIONS
-    -i, --input=input          Input Snyk Results JSON File
-    -o, --output=output        Output HDF JSON File
+convert snyk2hdf              Translate a Snyk results JSON file into a Heimdall
+                              Data Format JSON file
+                              A separate HDF JSON is generated for each project
+                              reported in the Snyk Report
+  USAGE
+    $ saf convert snyk2hdf -i <snyk-json> -o <hdf-scan-results-json> [-h]
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input Snyk Results JSON File
+    -o, --output=<value>  (required) Output HDF JSON File
 
   EXAMPLES
-    saf convert snyk2hdf -i snyk_results.json -o output-hdf-name.json
+    $ saf convert snyk2hdf -i snyk_results.json -o output-file-prefix
 ```
 
-
 ##### SonarQube to HDF
-
 ```
-convert sonarqube2hdf        Pull SonarQube vulnerabilities for the specified
-                             project name from an API and convert into a Heimdall
-                             Data Format JSON file
-  OPTIONS
-    -a, --auth=auth              SonarQube API Key
-    -u, --url=url                SonarQube Base URL (Excluding '/api')
-    -n, --projectKey=projectKey  SonarQube Project Key
-    -o, --output=output          Output HDF JSON File
+convert sonarqube2hdf         Pull SonarQube vulnerabilities for the specified
+                              project name and optional branch or pull/merge
+                              request ID name from an API and convert into a
+                              Heimdall Data Format JSON file
+  USAGE
+    $ saf convert sonarqube2hdf -n <sonar-project-key> -u <http://your.sonar.instance:9000> -a <your-sonar-api-key> [ -b <target-branch> || -p <pull-request-id> ] -o <hdf-scan-results-json>
 
-  EXAMPLES
-    saf convert sonarqube2hdf -n project_key -u http://sonar:9000 --auth YOUR_API_KEY -o output-hdf-name.json
+  FLAGS
+    -a, --auth=<value>           (required) SonarQube API Key
+    -b, --branch=<value>         Requires Sonarqube Developer Edition or above
+    -h, --help                   Show CLI help.
+    -n, --projectKey=<value>     (required) SonarQube Project Key
+    -o, --output=<value>         (required) Output HDF JSON File
+    -p, --pullRequestID=<value>  Requires Sonarqube Developer Edition or above
+    -u, --url=<value>            (required) SonarQube Base URL (excluding '/api')
 
+  EXAMPLES
+    $ saf convert sonarqube2hdf -n sonar_project_key -u http://sonar:9000 --auth YOUR_API_KEY [ -b my_branch || -p 123 ]-o scan_results.json
```

Comment:
just pick one or the other in the example.  provide a random string for the "YOUR API KEY" bit.  put a space before the output flag.